rasa/nlu/data_augmentation/augmenter.py

```diff
@@ -126,12 +149,18 @@ def _create_tokenizer_from_config(config_path: Text) -> Component:
     """
     config = rasa.shared.utils.io.read_config_file(config_path)
     pipeline = config.get("pipeline", [])
-    tokenizer_config = {"name": "WhitespaceTokenizer"}
+    tokenizer_config = {}
     for component in pipeline:
         if component.get("name", "").lower().endswith("tokenizer"):
             tokenizer_config = component
             break
 
```

Comment:
I just saw that there is a function which does exactly this - `_get_tokenizer_from_nlu_config` inside `story_conflict.py`. I think that one has a more full proof logic since it does not rely on the name of the component. The problem with relying on that name is that a component can still be a tokenizer and not have `tokenizer` at the end of its name.
I think we should move `_get_tokenizer_from_nlu_config` into a more shared utility module and reuse it. Wdyt?